src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -261,12 +261,22 @@ export const ComboBoxExample = {
         },
       ],
       text: (
-        <p>
-          To only allow the user to select a single option, provide the{' '}
-          <EuiCode>singleSelection</EuiCode> prop. You may want to render the
-          selected option as plain text instead of pill form. To do this, pass{' '}
-          <EuiCode>{'singleSelection={{ asPlainText: true }}'}</EuiCode>
-        </p>
+        <Fragment>
+          <p>
+            To only allow the user to select a single option, provide the{' '}
+            <EuiCode>singleSelection</EuiCode> prop. You may want to render the
+            selected option as plain text instead of pill form. To do this, pass{' '}
+            <EuiCode>{'singleSelection={{ asPlainText: true }}'}</EuiCode>
+          </p>
+          <p>
+            <strong>Note:</strong> <EuiCode>append</EuiCode> and{' '}
+            <EuiCode>prepend</EuiCode> props only works if
+            <EuiCode>singleSelection</EuiCode> prop is set to{' '}
+            <EuiCode>true</EuiCode> to avoid multilines that makes combobox
```

Comment:
Only because it can be true OR `asPlainText`, so it's just easier to mention the false state
```suggestion
            <EuiCode>singleSelection</EuiCode> prop is not set to{' '}
            <EuiCode>false</EuiCode> to avoid multilines that makes combobox
```